ui/src/components/item/QItem.js

```diff
@@ -58,7 +58,7 @@ export default createComponent({
       + (props.dense === true ? ' q-item--dense' : '')
       + (isDark.value === true ? ' q-item--dark' : '')
       + (
-        hasLink.value === true
+        hasLink.value === true && props.active === undefined
```

Comment:
`props.active` is guaranteed to be either `true` or `false` - never `undefined` (and `undefined` does not comply with repo's ESLint rules - use `void 0`)

You must change line 23 from `active: Boolean,` to

```javascript
active: {
  type: Boolean,
  default: null
}
```

Then use `&& props.active === null`

Reply:
Also you have to change the json definition for QItem

Reply:
Arg yes, indeed I forgot about this specificity of the `Boolean` vue type